src/controllers/notifications.js

```diff
@@ -1,67 +0,0 @@
-'use strict'
```

Comment:
It is kinda documented in the link I added above (https://github.com/interledger/five-bells-ledger/blob/ws_notifications/src/controllers/accounts.js#L126). Isn't it sufficient for now?